src/main/java/com/jagrosh/giveawaybot/commands/StartCommand.java

```diff
@@ -128,6 +129,13 @@ else if(list.size() >= level.maxGiveaways)
                     + (level.perChannelMaxGiveaways ? "channel" : "server") + "!");
             return;
         }
+
+        GuildSettings settings = bot.getDatabase().settings.getSettings(event.getGuild().getIdLong());
+        if (!level.canSetEmoji() && !settings.getEmojiDisplay().equals(Constants.TADA))
+        {
+            bot.getDatabase().settings.updateEmoji(event.getGuild(), null);
+            event.replyWarning("Your custom emoji has been reset, continuing with the giveaway.");
```

Comment:
I'm not a huge fan of having this additional message sent; this should either send a message and stop creation, tack the error message onto the result, or just not send anything